lib/zold/version_upgrade_manager.rb

```diff
@@ -0,0 +1,48 @@
+require 'semantic'
+
+module Zold
+  # Class to manage data upgrades (when zold itself upgrades).
+  class VersionUpgradeManager
+    # @todo #285:30min Refactor so we don't just assume there's .zoldata
```

Comment:
@botanicus I don't see the reasoning behind this... We will just have `version` file in the dir and that's it. What other situations may happen?